resolwe/flow/static/flow/processSchema.json

```diff
@@ -160,12 +171,45 @@
                             "type": "boolean"
                         }
                     }
+                },
+                "relations": {
+                    "description": "List of relations used by the process",
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "type": {
+                                "type": "string"
+                            },
+                            "category": {
+                                "type": "string"
+                            },
+                            "labels": {
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                }
+                            },
+                            "required": {
+                                "type": "boolean"
+                            }
+                        },
+                        "required": [
```

Comment:
Should `category` also be required?

Reply:
No, see this use case: https://github.com/genialis/resolwe-bio/pull/740/files#diff-8abd821c113e3327b5dc4e1038345c3eR14